apis/logging/v2beta/cluster_log_forwarder_types.go

```diff
@@ -0,0 +1,384 @@
+/*
```

Comment:
Good question - that's what I originally called it, but I switched to "beta" because it seems like we won't have a long trial period before GA so this will be "almost final". I am OK with either.

Reply:
@alanconway I do not recall the link, but you need to call it `v2beta1` then. K8S conventions require numbering for alpha and beta too.